win_configure/msi/wix/prep.bat

```diff
@@ -88,6 +88,7 @@ if %ERRORLEVEL% == 0 (
 		"%binaries_path%\tcltk\bin\*.dll"
 	) do (
 	xcopy /s /d "%%~a" "%EXECDIR%\bin" > NUL
+	if NOT %ERRORLEVEL% == 0 goto theend
 	)
 ) else (
 echo Fail to create %EXECDIR%\bin directory.
```

Comment:
Could you change this to "Failed to create .."? 
Also throughout the rest of the file.